contrib/download-frozen-image-v2.sh

```diff
@@ -80,6 +82,109 @@ fetch_blob() {
 	fi
 }
 
+# handle 'application/vnd.docker.distribution.manifest.v2+json' manifest
+handle_single_manifest_v2() {
+	local manifestJson="$1"; shift
+
+	configDigest="$(echo "$manifestJson" | jq --raw-output '.config.digest')"
+	imageId="${configDigest#*:}" # strip off "sha256:"
```

Comment:
Please use `local` for all variables introduced/local to this function, as in `fetch_blob`.